client/views/footer.jade

```diff
@@ -1,5 +1,15 @@
 template(name="footer")
-  footer(class="{{#if hideFooter}} hidden {{/if}}")
-    .container
-      .eha
-        a(href="http://ecohealthalliance.org")
+  footer
+    .eha
+      a.pull-left(href="http://ecohealthalliance.org")
+    .contact
+      .pull-right
+        a(href="mailto:eidr@ecohealthalliance.org") eidr@ecohealthalliance.org
+      .pull-right
+        .bullet
+      .pull-right
+        | 212.380.4460
+      .pull-right
+        .bullet
+      .pull-right
+        | 460 West 34th Street 17th Floor, New York, NY 10001-2320
```

Comment:
This can probably be handled by adding each contact method to a list item then changing the display property of the items to `inline-block`